packages/backend/src/document/document.controller.ts

```
@@ -97,6 +103,35 @@ export class DocumentController {
+    // Same read gate as GET /documents/:id — the file inherits the
+    // document's access policy.
+    await this.workspaceService.assertMember(
+      doc.workspaceId,
+      Number(req.user.id),
+    );
+    if (!doc.fileId || !VALID_FILE_ID_PATTERN.test(doc.fileId)) {
+      throw new NotFoundException('Document has no file');
+    }
+    const { body, contentType } = await this.fileService.getObject(
+      doc.fileId,
+    );
```

ESLint

[error] Replace `⏎······doc.fileId,⏎····` with `doc.fileId`

(prettier/prettier)